api.md

```diff
@@ -0,0 +1,86 @@
+# Proposed webrtc webextension api
+
+The initial version of the webrtc webextension api is a limited
+set of events, just enough to implement the extension from
+[this mozhacks article](https://hacks.mozilla.org/2015/09/controlling-webrtc-peerconnections-with-an-extension/).
+
+To use any of these events, an extension must have the `"webrtc"`
+permission (in addition to the `"experiments.webrtc"` permission unless/until
+this api lands in central).  In addition, an extension must have the
+`"webrtc.blocking"` permission to add a blocking onPeerConnection listener.
+
+The events are:
+
+## onPeerConnection
```

Comment:
The intention here is that there are two ways to use this event: you can just listen to it to know that a peer connection has been established, or you can create a blocking handler that gets to decide whether to accept or deny the connection.  The word "permission" seems to imply a particular scenario for blocking handlers.  Its probably a very common scenario, but I think it mis-characterizes what the event is...